File: client-app/src/app/models/service/serviceRate.ts
export interface ServiceRate {
    serviceRateId?: string;
    makeId: string;
    makeDescription?: string;
    modelId: string;
    modelDescription?: string;
    productStoreId?: string;
    productStoreName?: string;
    rate: number;
    fromDate: Date;
    thruDate?: Date;
}

File: Domain/ServiceRate.cs
namespace Domain;

public class ServiceRate
{
    public string ServiceRateId { get; set; }
    public string MakeId { get; set; }
    public string ModelId { get; set; }
    public string ProductStoreId { get; set; }
    public decimal? Rate { get; set; }
    public DateTime? FromDate { get; set; }
    public DateTime? ThruDate { get; set; }
    public DateTime? LastUpdatedStamp { get; set; }
    public DateTime? CreatedStamp { get; set; }


    public ProductCategory MakeProductCategory { get; set; }
    public ProductCategory ModelProductCategory { get; set; }
    public ProductStore ProductStore { get; set; }
}